Docs/SupportedAPIs.md

```diff
@@ -66,8 +66,8 @@ Contributions to expand support to unimplemented functionality are always welcom
 | GET    | `/session/:sessionId/window/size`                   | Supported    | Not implemented     |
 | POST   | `/session/:sessionId/window/:windowHandle/size`     | Supported    | `Session.resize()`  |
 | GET    | `/session/:sessionId/window/:windowHandle/size`     | Supported    | `Session.size()`    |
-| POST   | `/session/:sessionId/window/:windowHandle/position` | Supported    | Not implemented     |
-| GET    | `/session/:sessionId/window/:windowHandle/position` | Supported    | Not implemented     |
-| POST   | `/session/:sessionId/window/:windowHandle/maximize` | Supported    | Not implemented     |
+| POST   | `/session/:sessionId/window/:windowHandle/position` | Supported    | `Session.reposition()`|
```

Comment:
The last column needs updating for all window ops now, and we should remove `/session/:sessionId/window/size` since it doesn't exist.